projects/js-packages/components/components/pricing-table/styles.module.scss

```diff
@@ -0,0 +1,102 @@
+.container {
+	--padding: 32px;
+	color: var( --jp-black );
+}
+
+.table {
+	--gap: 24px;
+	position: relative;
+	z-index: 2;
+	padding: var( --padding ) 0;
+
+	.is-viewport-large & {
+		display: grid;
+		grid-template-columns: repeat( var( --columns ), 1fr );
+		grid-auto-flow: column;
+		grid-template-rows: repeat( var( --rows ), minmax( min-content, max-content ) );
+		column-gap: var( --gap );
+	}
+}
+
+.card {
+	background: var( --jp-white );
+	box-shadow: 0px 4px 24px rgba( 0, 0, 0, 0.05 ), inset 0px 0px 0px 1px var( --jp-gray );
+	border-radius: 4px;
+	padding: var( --padding ) 0;
+	margin-top: var( --padding );
+
+	& + & {
+		margin-top: var( --gap );
+	}
+}
+
+.header {
+	margin-bottom: var( --padding );
+	padding: 0 var( --padding );
+
+	.is-viewport-large & {
+		&::before {
+			top: 0;
+			position: absolute;
+			height: 100%;
+			width: calc(
+				( 100% / var( --columns ) ) - ( ( var( --columns ) - 1 ) * var( --gap ) / var( --columns ) )
```

Comment:
IMO this approach has 2 problems:

Is difficult to read all the calc and looks really complex and could break in some moment if something changes in the structure.

I took the freedom to create a new branch and suggest a refactor at this, using `display: contents`: 

https://github.com/Automattic/jetpack/commit/2c0c5a2afb7cc7c4cf5567d3e45e156a9959e6e6

Take a look at it, and let me know if makes sense.

Reply:
Should be better now!